houston/runner.py

```diff
@@ -32,14 +32,45 @@ def shutdown(self):
             self.__sandbox.destroy()
             self.__sandbox = None
 
+class MissionRunnerWithCoverage(threading.Thread):
```

Comment:
It looks like there's a lot of duplicated code in this class. Could `with_coverage` be a parameter of `MissionRunner` instead?

Reply:
I did it this way because of `files_to_instrument` but since it's no longer needed I can change it.

Reply:
Great.